zipline-kotlin-plugin/src/main/kotlin/app/cash/zipline/kotlin/AdapterGenerator.kt

```diff
@@ -704,14 +703,6 @@ internal class AdapterGenerator(
       )
     }
 
-    outboundServiceClass.addFakeOverrides(
```

Comment:
Okay, removing this fixed the duplicate function generation issue. Function overrides are already handled above and `equals`/`hashCode`/`toString` aren't bridge so I think this should be safe?